portfolio/src/main/webapp/style.css

```diff
@@ -1,108 +1,269 @@
-    justify-content: center;
+.header ul li a, .footer div a, .header h1 a {
+  display: inline-block;
+  margin: 0 8px;
+  text-decoration: none;
+  color: #fdf7f7;
+}
+
+.picture {
+  max-width: 25%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  border-radius: 20px;
+}
+
+.text-content {
     flex-direction: column;
-    text-align: center; 
-    padding: 50px; 
-    margin: 50px 50px 20px 50px;
-    background: #7b7b66;  
 }
 
-p{
-    font-size: 1.0em;
+.about ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: block;
 }
 
-#buttonline{
-    flex-direction: row;
+.description h1,
+.description p,
+.comment h1,
+.list-group-item {
+  color: #212529;
 }
 
+.description p,
+.skills p {
+  text-align: left;
+  font-size: 1.6rem;
+  padding-bottom: 3rem;
+}
 
-button, input[type='submit'] {
-    outline: none;
-    padding: 10px 10px;
-    font-size: 1.5em;
-    color: #c87e1e;
-    font-weight: bold;
-    background: #d3da82;
-    transition: all 0.2s ease-in-out;
-    margin-bottom: 1em;
+.about-text {
+  text-align: center;
+  font-size: 1.6rem;
+  padding-bottom: 3rem;  
 }
 
-.delete-button{
-    margin-left: 30px;
-    padding: 3px 3px;
-    font-size: 1em;
+.title {
+  text-align: center;
+  font-size: 2.3rem;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
 }
 
-.list-group-item{
-    color:black;
+.comments-list {
+  display: flex;
 }
 
-input{
-    width: 100%;
-    padding: 12px 20px;
-    margin: 8px 0;
-    display: inline-block;
-    border: 1px solid #ccc;
-    border-radius: 4px;
-    box-sizing: border-box;
+.myButton {
+  box-shadow: inset 0px 20px 25px -3px #f7ffe0;
+  background: linear-gradient(to bottom, #dbe6c4 5%, #9ba892 100%);
+  background-color: #dbe6c4;
+  border-radius: 6px;
+  border: 1px solid #b2b8ad;
+  display: inline-block;
+  cursor: pointer;
+  color: #757d6f;
+  font-family: Arial;
+  font-size: 15px;
+  font-weight: bold;
+  padding: 24px 76px;
+  text-decoration: none;
+  text-shadow: 0px 2px 0px #ced9bf;
 }
 
-#comments{
-    border-radius: 5px;
-    background-color: #f2f2f2;
-    padding: 20px;
+.myButton:hover {
+  background: linear-gradient(to bottom, #9ba892 5%, #dbe6c4 100%);
+  background-color: #9ba892;
 }
 
-.description {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-direction: column;
-    text-align: center; 
-    padding: 50px; 
-    margin-top: 0;
-    margin: 0px 50px 50px 50px;
-    background: #b4a68d;  
+.login {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 5rem;
+}
+
+.comments-list {
+  display: block;
+  width: 100%;
+}
+
+.list-group-item {
+  margin-bottom: 1rem;
+  list-style: none;
+  background: beige;
+  padding: 1rem;
+  border-radius: 10px;
+  box-shadow: 2px 2px 2px gainsboro;
+  text-align: left;
+}
+
+.user {
+  font-weight: bold;
+}
+
+.delete-button {
+  float: right;
+  font-size: 15px;
+  border: none;
+  font-weight: 400;
+  border-radius: 6px;
```

Comment:
You should declare a base button class to avoid repeating common properties between buttons.

You could have one class called "button", and others called "warning-button", etc.

You can take some inspirations from here: https://material.angular.io/components/button/overview.

Same comment applies for input fields, try to use more generic names for basic styling of components.